baseclasses/chemical_energy/potentiometry_gaschromatography.py

```diff
@@ -0,0 +1,478 @@
+                    'AEM',
+                    'CEM'
+                ])))
+
+    membrane_name = Quantity(
+        type=str,
+        shape=[],
+        a_eln=dict(
+            component='EnumEditQuantity',
+            props=dict(
+                suggestions=[
+                    'PiperION',
+                    'SustainION',
+                    'Fumasep',
+                    'Nafion'
+                ])))
+
+    membrane_thickness = Quantity(
+        type=np.dtype(np.float64),
+        unit=('um'),
+        a_eln=dict(component='NumberEditQuantity', defaultDisplayUnit='um', props=dict(minValue=0)))
+
+    gasket_thickness = Quantity(
+        type=np.dtype(np.float64),
+        unit=('um'),
+        a_eln=dict(component='NumberEditQuantity', defaultDisplayUnit='um', props=dict(minValue=0)))
+
+    anolyte_type = Quantity(
+        type=str,
+        shape=[],
+        a_eln=dict(
+            component='EnumEditQuantity',
+            props=dict(
+                suggestions=[
+                    'KOH',
+                    'KHCO3'
+                ])))
+
+    anolyte_concentration = Quantity(
+        type=np.dtype(np.float64),
+        unit=('mol/L'),
+        a_eln=dict(component='NumberEditQuantity', defaultDisplayUnit='mol/L'))
+
+    anolyte_flow_rate = Quantity(
+        type=np.dtype(np.float64),
+        unit=('ml/minute'),
+        a_eln=dict(
+            component='NumberEditQuantity',
+            defaultDisplayUnit='ml/minute',
+            props=dict(minValue=0)
+        ))
+
+    anolyte_volume = Quantity(
+        type=np.dtype(np.float64),
+        unit=('ml'),
+        a_eln=dict(component='NumberEditQuantity', defaultDisplayUnit='ml'))
+
+    has_humidifier = Quantity(
+        type=bool,
+        default=True,
+        a_eln=dict(component='BoolEditQuantity')
+    )
+
+    humidifier_temperature = Quantity(
+        type=np.dtype(np.float64),
+        unit='°C',
+        a_eln=dict(component='NumberEditQuantity', defaultDisplayUnit='°C'))
+
+    water_trap_volume = Quantity(
+        type=np.dtype(np.float64),
+        unit=('ml'),
+        a_eln=dict(component='NumberEditQuantity', defaultDisplayUnit='ml'))
+
+    feed_gases = SubSection(
+        section_def=NECCFeedGas, repeats=True)
+
+    bleedline_flow_rate = Quantity(
+        type=np.dtype(np.float64),
+        unit=('ml/minute'),
+        a_eln=dict(
+            component='NumberEditQuantity',
+            defaultDisplayUnit='ml/minute',
+            props=dict(minValue=0)
+        ))
+
+    nitrogen_start_value = Quantity(
+        type=np.dtype(np.float64),
+        description='Specified in ppm',
+        a_eln=dict(component='NumberEditQuantity'))
+
+    # TODO Should an empty option also be specified? If excel is read and CP/CA is not set, CP is set automatically.
+    chronoanalysis_method = Quantity(
+        type=MEnum('Chronoamperometry (CA)', 'Chronopotentiometry (CP)'),
+        shape=[],
+        a_eln=dict(component='EnumEditQuantity'))
```

Comment:
Should an empty option be given in the enum for the chronoanalysis_method? Or how to handle enum when empty field is parsed in excel?

Reply:
The alternative is to make it a string with suggestions:

```python
method = Quantity(
        type=str,
        shape=[],
        a_eln=dict(
            component='EnumEditQuantity',
            props=dict(
                suggestions=['XRR', 'XRD', 'XRF', 'Ellipsometry', 'Sputtering', 'SEM_Merlin', 'Measurement', 'Synthesis', 'Dektak', 'TGA', "PECVD", "Catalytic_Reaction"])
        ))
```
or an empty option

Reply:
Yes, I remember using the suggestions also at other places. Thank you.